wordpress-mobile/WordPress-Android
# Shorter sign up - remove Send Link by email screen

Fixes #13971
Relates [WordPressAuthenticator-iOS#574](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/574)

This PR removes the below intermediate `Sign Up` screen:

<img src="https://user-images.githubusercontent.com/9729923/107496782-6ba45700-6b9a-11eb-8afd-9ce685baaecd.png" width="250" height="500">

Instead, an email will be now sent immediately after the user enters their email:

https://user-images.githubusercontent.com/9729923/107495717-203d7900-6b99-11eb-94bb-6393162595e3.mp4

ℹ️ An additional paragraph was added on the confirmation screen stating the: `Didn't mean to create a new account? Go back to re-enter your email address.`

⚠️ Since the `gotUnregisteredEmail(...)` functionality is shared between the `WordPress` and `WooCommerce` apps, this was not removed altogether. Instead the implementing client just end up doing nothing as an implementation on the `WordPress` side.

To test:
- Verify that the login and signup flows are working as expected, with no issue and with the above flow improvement.
- Check tracking is as expected.

PR submission checklist:
